src/server/scripts/Spells/spell_generic.cpp

```diff
@@ -4595,6 +4595,74 @@ class spell_gen_impatient_mind : public SpellScriptLoader
         }
 };
 
+enum SlagBehemothSpells
+{
+    SPELL_BLAST_WAVES_ONE_TICK      = 159749,
+    SPELL_BLAST_WAVES_THREE_TICK    = 159760
+};
+
+// Called by 159750: TickNumber < 3 should call 159749, TickNumber == 3 should called 159760 and destroy volcanic bomb.
+class spell_gen_slag_behemoth_blast_waves : public SpellScriptLoader
+{
+public:
+    spell_gen_slag_behemoth_blast_waves() : SpellScriptLoader("spell_gen_slag_behemoth_blast_waves") { }
+
+    class spell_gen_slag_behemoth_blast_waves_AuraScript : public AuraScript
+    {
+        PrepareAuraScript(spell_gen_slag_behemoth_blast_waves_AuraScript);
+
```

Comment:
Missing validate

Reply:
@Ulduar2

Reply:
You can close this request. I don't have instrument on current state for check script.
Or you can redone this.